server/utils/validation.ts

```diff
@@ -1,22 +1,41 @@
 import type { Request, Response } from 'express'
 import jsonpath from 'jsonpath'
 
-import type {
-  ErrorMessage,
-  ErrorMessages,
-  ErrorSummary,
-  ErrorsAndUserInput,
-  ErrorsTitleAndUserInput,
-} from '@approved-premises/ui'
+import type { ErrorMessage, ErrorMessages, ErrorSummary, ErrorsAndUserInput } from '@approved-premises/ui'
 import { SanitisedError } from '../sanitisedError'
 import errorLookup from '../i18n/en/errors.json'
 import { TasklistAPIError, ValidationError } from './errors'
+import { generateConflictBespokeError } from './bookingUtils'
 
 interface InvalidParams {
   propertyName: string
   errorType: string
 }
 
+export const generateConflictErrorAndRedirect = (
```

Comment:
Yep, like this approach!

Reply:
I guess the downside is validation.ts knowing about this specific error type, but 🤷‍♀️